backend/ml/inputs.py

```diff
@@ -69,35 +67,33 @@ def get_comparisons(
 
     @cached_property
     def ratings_properties(self):
-        # TODO support trust_scores from the public dataset
         user_entities_pairs = pd.Series(
             iter(
                 set(self.public_dataset.groupby(["user_id", "entity_a"]).indices.keys())
-                | set(
-                    self.public_dataset.groupby(["user_id", "entity_b"]).indices.keys()
-                )
+                | set(self.public_dataset.groupby(["user_id", "entity_b"]).indices.keys())
             )
         )
         dtf = pd.DataFrame([*user_entities_pairs], columns=["user_id", "entity_id"])
         dtf["is_public"] = True
         top_users = dtf.value_counts("user_id").index[:6]
```

Comment:
Why don't we set `top_users = dtf.value_counts("user_id").index[:MAX_SCALING_CALIBRATION_USERS]` ?

Reply:
Actually, the top_users also need I guess a trust score higher than 0.1.

Reply:
I updated `MlInputFromPublicDataset` to use the same parameters for scaling calibration. Hopefully, it will make things clearer, even if `MlInputFromPublicDataset` is not used for now.